trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './decorator'
+
+# A class to trim names
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
+  end
```

Comment:
- [ ] As requested by the previous reviewer please make sure to implement the trimmer decorator class as if the name has a maximum of 10 characters. If it's longer it should trim the word.
```ruby
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```
what does the code do `the correct_name method will return a corrected version of the name contained in @nameable by either truncating it to 10 characters (if the original name is longer than 10 characters) or returning the original name (if it is 10 characters or less).`